docs/package.json

```diff
@@ -14,19 +14,20 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "3.1.1",
-    "@docusaurus/preset-classic": "3.1.1",
-    "@docusaurus/theme-mermaid": "^3.1.1",
+    "@docusaurus/core": "0.0.0-5872",
```

Comment:
Do we wanna wait until this fix is in a stable release? Deadline would be the 1.18.0 if it is not released by then we can go ahead and merge this

Reply:
I would lean towards merging, as it would allow to work on things like optimizing SEO in parallel.
Given that this dependency is `docs` specifics, I think it's fine.